projects/frontend-ui/shared/README.md

```diff
@@ -0,0 +1,17 @@
+# VDK Shared Components library
+
+This project implements common UI functionality that is used in the data-pipelines project.
+Everything in this folder is project agnostic and there is nothing data-pipelines specific about them.
```

Comment:
the "shared" shouldn't know about data-pipelines (since agnostic), I would even keep that out of the shared README. It's just meant to be reused.